Common/src/linear_solvers_structure.cpp

```diff
@@ -87,9 +87,12 @@ void CSysSolve::SolveReduced(const int & n, const vector<vector<su2double> > & H
 void CSysSolve::ModGramSchmidt(int i, vector<vector<su2double> > & Hsbg, vector<CSysVector> & w) {
   
   bool Convergence = true;
+  
+#ifdef HAVE_MPI
```

Comment:
Could you please remove the ifdef's? These should be unnecessary now that the wrapper takes care of it internally. Or was there a problem?